build-system/pr-check.js

```diff
@@ -225,6 +225,11 @@ const command = {
     // Disabled because it regressed. Better to run the other saucelabs tests.
     // execOrDie(`${gulp} test --nobuild --saucelabs --oldchrome --compiled`);
   },
+  runVisualDiffTests: function() {
+    // This must only be run for push builds, since Travis hides the encrypted
+    // environment variables required by Percy during pull request builds.
+    execOrDie(`${gulp} visual-diff`);
```

Comment:
FYI, the actual percy invocation in the visual-diff.js task was already wrapped in an "or-don't-die", but I see no harm in making sure pr-check.js doesn't terminate on failure either. I've got a bug assigned to me to refactor all these exec* functions into one library.